### PR TITLE
DB-8779 update the system view SYSCONGLOMERATEINSCHEMAS to project ou…

### PR DESCRIPTION
**Pull request by yxia92**
…t the column CONGLOMERATENAME

### CODE REVIEW

**Comment by yxia92** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by yxia92** [comment]
jenkins please retest branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by yxia92** [comment]
jenkins please retest branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java [hunk 1]
```diff
@@ -1410,6 +1410,32 @@ public void updateSystemSchemasView(TransactionController tc) throws StandardExc
...
+        // check sysconglomerateinschemas view
+        TableDescriptor td = getTableDescriptor("SYSCONGLOMERATEINSCHEMAS", sysVWSchema, tc);
+        if (td != null) {
+            ViewDescriptor vd = getViewDescriptor(td);
+            boolean needUpdate = !vd.getViewText().equals(SYSCONGLOMERATESRowFactory.SYSCONGLOMERATE_IN_SCHEMAS_VIEW_SQL);
+
+            // view definition matches the setting, no update needed
+            if (!needUpdate) return;
+
+            // drop the view deifnition
```
Comment (arnaud-splice) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java]: [minor] deifnition -> definition

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java [hunk 2]
```diff
@@ -1410,6 +1410,32 @@ public void updateSystemSchemasView(TransactionController tc) throws StandardExc
...
+            dropAllColumnDescriptors(td.getUUID(), tc);
+            dropViewDescriptor(vd, tc);
+            dropTableDescriptor(td, sysVWSchema, tc);
+
+        }
+
+        // add new view deifnition
```
Comment (arnaud-splice) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java]: [minor] deifnition -> definition

**Pull request merged**